src/gen_pfc.c

```diff
@@ -117,8 +117,11 @@ static void _pfc_arg(FILE *fds,
  */
 static void _pfc_action(FILE *fds, uint32_t action)
 {
-	switch (action & 0xffff0000) {
-	case SCMP_ACT_KILL:
+	switch (action & SECCOMP_RET_ACTION_FULL) {
+	case SCMP_ACT_KILL_PROCESS:
+		fprintf(fds, "action KILL-PROCESS;\n");
```

Comment:
Nitpick: since the #define uses an underscore, let's use an underscore here:

    fprintf(fds, "action KILL_PROCESS;\n");